cms/static/js/features_jsx/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
[eslint-config-airbnb-base](https://www.npmjs.com/package/eslint-config-airbnb-base) doesn't contain any rules for linting jsx syntax. [eslint-config-airbnb](https://www.npmjs.com/package/eslint-config-airbnb-base) extends eslint-config-airbnb-base but also includes rules for linting jsx. Brian used eslint-config-airbnb-base instead of eslint-config-airbnb because we had no need for the jsx linting at the time that he implemented it, but now we will need to switch to enable full support.